### PR TITLE
Add font on mui theme and update risk page

### PR DESCRIPTION
**Pull request by nkrulikovsky**
- Add font on MUI theme
- Update Risk page based on the font family added and MUI component.

## Screenshot

Screen Shot 2022-01-10 at 7 14 34 PM

### CODE REVIEW

**Review by ug02fast — APPROVED** [review]
exciting

**Comment by nkrulikovsky** [comment]
We will move font folder to theme when we don't use SCSS.

File: public/index.html [hunk 1]
```diff
@@ -2,27 +2,41 @@
...
+    <meta name="msapplication-TileColor" content="#da532c" />
     <meta name="theme-color" content="#ffffff" />
-    <meta name="ui-version" content="%REACT_APP_GIT_SHA%">
+    <meta name="ui-version" content="%REACT_APP_GIT_SHA%" />
+
+    <!-- Source code pro font  -->
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Source+Code+Pro:ital,wght@0,200;0,300;0,400;0,500;0,800;0,900;1,200;1,300;1,400;1,500;1,700;1,800;1,900&family=Work+Sans:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
```
Comment (hammeiam) [inline public/index.html:36]: do we need this many weights in both normal and italic? Could we get away with 200, 500, 800?
Is Work-Sans used anywhere or can it be removed?

Comment (nkrulikovsky) [reply public/index.html]: Just confirmed that we don't use much font on our theme and removed unnecessary font family.

**Review by hammeiam — APPROVED** [review]

**Pull request merged**